boostlook.css

```diff
@@ -35,78 +35,107 @@
 /*----------------- Root Variables - Start -----------------*/
 
 :root {
-  --light-bl-pasteboard-color: rgb(209, 213, 219);
-  --light-bl-pre-background: rgb(248, 248, 248);
-  --light-bl-table-border-color: rgb(232, 232, 232);
-  --light-bl-table-head-background: rgb(248, 248, 248);
+  --bl-primary-orange: rgb(255, 159, 0);
```

Comment:
primary, secondary, etc. colors shouldn't have the color as part of the name, as it could change. --bl-primary-color is fine

Reply:
Resolved in latest commit